src/macros.jl

```diff
@@ -446,19 +446,28 @@ function constraint_macro(args, macro_name::Symbol, parsefun::Function)
         $parsecode
         $(refcall) = $constraintcall
     end
+
     # Determine the return type of addconstraint. This is needed for JuMP extensions for which this is different than ConstraintRef
     contype = :( constrainttype($m) )
+    creationcode = getloopedcode(variable, code, condition, idxvars, idxsets, contype, requestedcontainer)
+
+    if anonvar
+        # Anonymous constraint, no need to register it to a name
```

Comment:
Clarify that "name" here refers to the model-level dictionary and not the string name.